fab/fabfile.py

```diff
@@ -560,10 +560,16 @@ def awesome_deploy(confirm="yes", resume='no'):
         utils.abort('Deployment aborted.')
 
     if resume == 'yes':
-        env.update(retrieve_cached_deploy_env())
+        try:
+            cached_payload = retrieve_cached_deploy_env()
+            checkpoint_index = retrieve_cached_deploy_checkpoint()
+        except Exception:
+            print red('Unable to resume deploy, please start anew')
```

Comment:
might want to add in a suggestion to delete the `.pickle` files so they can resume future deploys (or will that just be overwritten anyways)?